src/application/commands/create-suggestion-du-conseiller.command.handler.ts

```diff
@@ -0,0 +1,112 @@
+import { CommandHandler } from '../../building-blocks/types/command-handler'
+import { emptySuccess, Result } from '../../building-blocks/types/result'
+import { Authentification } from '../../domain/authentification'
+import { AuthorizeConseillerForJeunes } from '../authorizers/authorize-conseiller-for-jeunes'
+import { Recherche } from '../../domain/offre/recherche/recherche'
+import Suggestion = Recherche.Suggestion
+import { DateService } from '../../utils/date-service'
+import { SuggestionsRepositoryToken } from '../../domain/offre/recherche/suggestion/suggestion'
+import { Inject } from '@nestjs/common'
+import { IdService } from '../../utils/id-service'
+
+export interface CreateSuggestionDuConseillerCommand {
+  idJeunes: string[]
+  type: Recherche.Type
+  criteres: Recherche.Emploi
+}
+
+export class CreateSuggestionDuConseillerCommandHandler extends CommandHandler<
+  CreateSuggestionDuConseillerCommand,
+  void
+> {
+  constructor(
+    private conseillerForJeunesAuthorizer: AuthorizeConseillerForJeunes,
+    @Inject(SuggestionsRepositoryToken)
+    private suggestionRepository: Suggestion.Repository,
+    private dateService: DateService,
+    private idService: IdService
+  ) {
+    super('CreateSuggestionDuConseillerCommandHandler')
+  }
+
+  async authorize(
+    command: CreateSuggestionDuConseillerCommand,
+    utilisateur: Authentification.Utilisateur
+  ): Promise<Result> {
+    return this.conseillerForJeunesAuthorizer.authorize(
+      command.idJeunes,
+      utilisateur
+    )
+  }
+
+  async handle(
+    command: CreateSuggestionDuConseillerCommand
+  ): Promise<Result<void>> {
+    const maintenant = this.dateService.now()
+
+    for (const idJeune of command.idJeunes) {
+      const idSuggestion = this.idService.uuid()
+      const idFonctionnel: Suggestion.IdFonctionnel =
+        construireIdFonctionnel(command)
+      const informations = construireInformationsSuggestion(
+        idFonctionnel,
+        command.criteres
+      )
+      const suggestion: Suggestion = {
+        dateCreation: maintenant,
+        dateRafraichissement: maintenant,
+        id: idSuggestion,
+        idFonctionnel,
+        idJeune,
+        type: command.type,
+        source: Suggestion.Source.CONSEILLER,
+        informations,
+        criteres: command.criteres
+      }
+
+      // TODO save si (jeune par jeune):
+      // TODO - Suggestion pas déjà proposée
+      // TODO - Suggestion pas déjà traitée
+      await this.suggestionRepository.save(suggestion)
```

Comment:
Before it goes too far : on a un PoleEmploiSuggestionService.raffraichir qui fait un peu exactement ce dont on a besoin n'est il pas. Il est.